Automattic/wp-calypso
# Change Expert Guidance offer image

#### Changes proposed in this Pull Request

* Change the offer image for the expert guidance offer introduced in https://github.com/Automattic/wp-calypso/pull/42783

#### Testing instructions



* Proxy through US IP, assign yourself to _variantShowOffer_ of whiteGloveUpsell A/B test.
* Go through the signup flow and select the free plan.
* In the offer page, verify that the image has been updated.
